src/components/common/button.js

```diff
@@ -26,14 +33,15 @@ const styleForVariant = variant =>
     dark: darkStyles,
     "light-grey": lightGreyStyles,
     "light-blue": lightBlueStyles,
+    borderless: borderlessStyles,
   }[variant] ||
   (() => {
     throw new Error(`Unknown button variant: ${variant}`);
   }));
 
 Button.propTypes = {
   variant: PropTypes.string.isRequired,
-  label: PropTypes.string.isRequired,
+  label: PropTypes.any.isRequired,
```

Comment:
looking more and more like typescript would help here, with an `enum` for `variant` and this `label` can be `string | whateverElse` (although it looks like it is only `string` so far anyway)

Reply:
yep, I'll definitely go through these and convert them to ts this week along with tailwind removal